thirdparty/cli-utils/status/printers/list/base.go

```diff
@@ -80,15 +84,50 @@ func (ep *BaseListPrinter) printStatusEvent(se pollevent.Event) error {
 	switch se.Type {
 	case pollevent.ResourceUpdateEvent:
 		id := se.Resource.Identifier
-		return ep.Formatter.FormatStatusEvent(event.StatusEvent{
-			Identifier:       id,
-			Resource:         se.Resource.Resource,
-			PollResourceInfo: se.Resource,
-		})
+		var invName string
+		var ok bool
+		if invName, ok = ep.Data.InvNameMap[id]; !ok {
+			return fmt.Errorf("Resource not found\n")
+		}
+		// filter out status that are not assigned
+		statusString := se.Resource.Status.String()
+		if _, ok := ep.Data.StatusSet[strings.ToLower(statusString)]; len(ep.Data.StatusSet) != 0 && !ok {
+			return nil
+		}
+		switch ep.Format {
+		case printers.EventsPrinter:
+			_, err := fmt.Printf("%s/%s/%s/%s is %s: %s\n", invName,
```

Comment:
`fmt.Printf` prints to `os.Stdout` directly. However, the underlying `cli-utils` library prints to `ioStreams.Out`, which is `os.Stdout` in most cases, but it can also be other streams.

I guess we need to wait for @mortent on how to proceed with the deviated fork. If we decide to go with the major changes in the forked cli-utils, it's better to pass along `ioStreams` for printing.